src/libs/Clipboard/index.js

```diff
@@ -1,4 +1,38 @@
 // on Web/desktop this import will be replaced with `react-native-web`
+import _ from 'lodash';
 import {Clipboard} from 'react-native-web';
 
-export default Clipboard;
+/**
+ * Writes the content as rich types if provided and the web client supports it, otherwise as a string.
+ * @param {Object} content Platfrom-specific content to write
+ * @param {String} content.text Text representation of the content for web
+ * @param {String} [content.html] Html representation of the content for web
+ */
+const setContent = (content) => {
+    if (!content) {
+        return;
+    }
+
+    // If HTML is present, try to write rich types.
+    if (content.html) {
+        if (!_.get(navigator, 'clipboard.write')) {
+            Clipboard.setString(content.text);
```

Comment:
```suggestion
```
Also, I think we should fallback to MD instead of text when this is not supported.

Reply:
We can convert to MD here, given that we do have HTML but cannot write it. Otherwise, you would have to generate and provide MD in all cases, even if you write HTML correctly for the platform. Even more importantly, the selection object signature has to work for links and other text, so if we added a markdown field, it could not be marked as mandatory, which defies the intent to be a fallback field.

Reply:
What do you think of my next comment?

Reply:
If we force the callee to do the conversion, then we can just throw an error and avoid writing twice. Or we can do it implicitly and convert it here without throwing an error. When the callee supplies HTML, technically we have all the input for all the platforms.

Reply:
It seems like your first point is the same as my suggestion https://github.com/Expensify/App/pull/9880#discussion_r927906326. The second is also doable. MY motive behind this change was to take the MD part out of setContent. Let's consider setContent as setter for HTML and text. You can also rename this to setHTML `setHTML(html,text)`.  

So which one would you like to do?

Reply:
Maybe another way is to have something like Clibpoard.canWriteHtm(), so the caller can detect what to supply without reinventing the wheel or catching an error.

Reply:
That's the same thing IMO. OK, so you go ahead and make some changes and we can see how it looks.